100-CNAB.md

```diff
@@ -29,7 +29,7 @@ The canonical encoding of a bundle definition is a JSON-formatted file, which MU
 
 In either case, CNAB has the same schema, and this spec refers to this file as the "bundle definition" (or occasionally "bundle file"). 
 
-However, as a signed bundle definition represents an immutable bundle, all invocation images and images references must have a digest.
+However, as a signed bundle definition represents an immutable bundle, all invocation images and images references must have a content-digest.
```

Comment:
or have a digested image reference (e.g.: myimage@sha256:aabb...)

Reply:
I disagree here. At the point at which a bundle is ready to be shared or shipped, a bundle should have a content digest regardless of whether the image reference contains a digest or not. Do you disagree?